docs/integrations/enterprise-connectors/source-oracle.md

```diff
@@ -69,11 +94,36 @@ GRANT SELECT ON "<schema_a>"."<table_1>" TO airbyte;
 GRANT SELECT ON "<schema_b>"."<table_2>" TO airbyte;
 ```
 
-Your database user should now be ready for use with Airbyte.
+Your database user should now be ready for use with Airbyte, except for CDC incremental syncs.
+In that case, your user needs an additional set of permissions.
```

Comment:
The best thing may be to replace these two lines with a new heading. This is easier for scanning content. For example:

`#### 4. If using CDC incremental syncs, grant your user extra permissions`